modules/enabling-OVS-balance-slb-mode.adoc

```diff
@@ -0,0 +1,208 @@
+// Module included in the following assemblies:
+//
+// IPI
+// * installing/installing_bare_metal/ipi/ipi-install-installation-workflow.adoc
+// * installing/installing_bare_metal/ipi/ipi-install-post-installation-configuration.adoc
+// UPI
+// * installing/installing_bare_metal/upi/installing-bare-metal-network-customizations.adoc
+// * installing/installing_bare_metal/upi/installing-restricted-networks-bare-metal.adoc
+// * installing/installing_bare_metal/upi/installing-bare-metal.adoc
+
+ifeval::["{context}" == "ipi-install-post-installation-configuration"]
+:postinstall-bare-metal-ipi:
+endif::[]
+
+:_mod-docs-content-type: PROCEDURE
+[id="enabling-OVS-balance-slb-mode_{context}"]
+= Enabling OVS balance-slb mode for your cluster
+
+You can enable the Open vSwitch (OVS) `balance-slb` mode on infrastructure where your cluster runs so that two or more physical interfaces can share their network traffic. A `balance-slb` mode interface provides source load balancing (SLB) capabilities for a cluster that runs virtualization workloads, where the interface can act independently without needing to communicate with a network switch.
+
+Source load balancing works by assigning a Media Access Control (MAC) address and a virtual local area network (vLAN), if required, to a bond interface, such as `br-phy`. Because of the shared MAC address and vLAN between interfaces, using `balance-slb` mode to share pod traffic has no benefit.
+
+The following diagram depicts `balance-slb` mode on a simple cluster infrastructure layout. Virtual machines (VMs) connect to a specific localnet `NetworkAttachmentDefinition` (NAD) custom resource definitions (CRDs), `NAD 0` or `NAD 1`. Each NAD provides VMs with access to network information, such as VLAN ID tags. A `br-ex` interface, the virtual switch, receives information from VMs and passes the information to an OVS bridge, `br-phy`. This bridge connects operates as a port to receive packets from a physical interface, such as `eno0`, and then transmits the package to another physical interface, such as `eno1`. Additionally, information from either physical interface can pass through the set of OVS bridges to reach the VMs.
```

Comment:
Each NAD provides VMs with access to one or more underlay networks via specifying VLAN ID tags. 

I would substitute the words "information" with "traffic" , and the word "package" with "packet.

A `br-ex` OVS bridge receives traffic from VMs and passes  to an OVS bridge, `br-phy`. The `br-phy` bridge functions as the controller for the slb-balanced bond. It balances traffic from different VM ports over the physical links `eno0` and `eno1`. Additionally, ingress traffic from either physical interface can pass through the set of OVS bridges to reach the VMs.

Reply:
:heavy_check_mark: